src/styles/index.scss

```diff
@@ -208,3 +208,8 @@ code {
 
 // Override with custom color styles
 @import './components';
+
+//Atlas icons import
+@import '~@vectopus/atlas-icons/style.css';
```

Comment:
@Samsunggora It would be convenient to remove the coreui import and try it without it. lets see what's broken.

Also, remove coreui dependency from demo package.json